### ISSUE TITLE
Left aligned image at the bottom causes prev & next buttons to jump up

### ISSUE THREAD

**Issue by joonaspaakko** [retypeapp/retype#232]
My fix for now: 
```md
-![](/images/my-image.gif)

<div style="clear: both;"></div>
```
What the issue looks like (text is blurred on purpose):

![retype-left-aligned-image-at-the-bottom](https://user-images.githubusercontent.com/1164476/144610931-a77bab8e-76c4-40d7-b1ad-bbfd7d3a4ac4.png)

**Comment by geoffreymcgill** [comment]
Thanks for reporting this issue. Should be an easy fix for us to prevent the Previous|Next buttons from interacting with any float elements above. We will try and get this fix into the `v1.12.0` release.

**Comment by geoffreymcgill** [comment]
This issue has been fixed and will be included in the upcoming `Retype v2.0` release. No official release date yet, but we are getting close.

Hope this helps.

**Comment by geoffreymcgill** [comment]
@joonaspaakko Your `<div style="clear: both;"></div>` workaround will not be required after upgrading to `Retype v2.0`.
